en/docs/references/extend/rebranding/customizable-login-portal.md

```diff
@@ -0,0 +1,142 @@
+# Customize login portal
```

Comment:
Have we used sentence case in other IS docs?
Let's use whatever is consistent

Reply:
I'll update this we usually use the camel case.